modelforge/potential/parameters.py

```diff
@@ -46,6 +46,16 @@ class ActivationFunctionParamsNegativeSlope(BaseModel):
     negative_slope: Optional[float] = None
 
 
+class AtomicNumber(BaseModel):
+    maximum_atomic_number: int = 101
```

Comment:
I thought we had decided on not including default values in the models.

Reply:
Removed the default values. I will note that `maximum_atomic_number` does not impact the outcome of the calculations since it effectivly says 'what is the largest one-hot encoding that I will allow', which is in itself not immpacting downstream predictions. In other words, I think this is one of the things that we could set a default value for convencience.